src/app/restaurantcard/restaurantcard.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { Restaurant } from '../shared/Models/restaurant';
+
+@Component({
+  selector: 'app-restaurantcard',
```

Comment:
Restaurant card is two words so they should be separated with a dash. `app-restaurant-card`

Reply:
This also goes for the file and folder name of the component